evalml/tests/automl_tests/test_automl.py

```diff
@@ -1521,30 +1527,49 @@ def test_describe_pipeline_with_ensembling(
 
     ensemble_ids = [
         _get_first_stacked_classifier_no() - 1,
+        _get_first_stacked_classifier_no(),
+        len(automl.results["pipeline_results"]) - 2,
         len(automl.results["pipeline_results"]) - 1,
     ]
 
+    num_sklearn_pl = 0
     for i, ensemble_id in enumerate(ensemble_ids):
+        sklearn_pl = (
+            True if "Sklearn" in automl.get_pipeline(ensemble_id).name else False
+        )
         caplog.clear()
         automl_dict = automl.describe_pipeline(ensemble_id, return_dict=return_dict)
         out = caplog.text
-        assert "Sklearn Stacked Ensemble Classification Pipeline" in out
+        if sklearn_pl:
+            assert "Sklearn Stacked Ensemble Classification Pipeline" in out
+            assert "* final_estimator : None" in out
+            num_sklearn_pl += 1
```

Comment:
Why are we adding 1 here? 👀 Do we use this variable anywhere else? / do we want to check the value of this?

Reply:
Forgot an assert, I'll add it at the end.